warc_metadata_sidecar.py

```diff
@@ -141,6 +154,8 @@ def create_string_payload(mime_dict, puid, result_dict, lang_cld):
         payload.append('{0} {1}'.format(CHARSET_TITLE, result_dict))
     if lang_cld:
         payload.append('{0} {1}'.format(LANGUAGE_TITLE, lang_cld))
+    if soft404:
```

Comment:
We may need to be more specific here if we want to be able to include the `Soft-404-Detected` field in the theoretical case that the probability is `0.0`
```suggestion
    if soft404 is not None:
```